### DISCUSSION TITLE
VMWare Workstation drag/drop not working with EOS 7

### DISCUSSION THREAD

**Discussion by rwoolf** [elementary/triage#328] [Q&A]
I have used EOS in VMWare Workstation for years.  With version 7, drag/drop, copy/paste no longer works with the host (Windows).  

Does anyone know how to get this working again?

**Comment by RogerWeihrauch** [comment]
hi
well, from give information I can only think of:
1) did you reinstall the VMware tools after going to your version of EOS?
2) did you try if C&P works in full screen mode of the guest?
3) is there an option in client's settings area in VMware Workstation to check if C&P is still enabled for this client?
Regards
Roger

**Reply by rwoolf** [reply]
I installed VMWare tools after installing a fresh EOS 7.  When it did not work, I reinstalled VMWare tools, still with no luck.  I generally work full screen with guests, so I have tried that and not as full screen.  I have disabled and reenabled C&P and D&D to see if that would reset anything, all with no luck.  I have read that for Ubuntu, if you have Wayland enabled instead of X that it won't work, so they say switch back to X.  But as I understand it EOS 7 is not using Wayland out of the box and I have not done anything to enable it.  I still have some old EOS 2 VMs that C&P and D&D work flawlessly, and I have some EOS 5 that are buggy when doing C&P and D&D but is still usable.  But with EOS 7 it does not work at all.
